rules/react/walmart.js

```diff
@@ -16,15 +16,15 @@ module.exports = {
     // Enforce boolean attributes notation in JSX
     "react/jsx-boolean-value": 2,
     // Validate closing bracket location in JSX
-    "react/jsx-closing-bracket-location": [2, 'tag-aligned'],
+    "react/jsx-closing-bracket-location": [2, "tag-aligned"],
```

Comment:
It's a style rule so "reasons" will be opinions. We could change it but if you're volunteering to update all the repos :).